src/nft/utils/fetchPrice.ts

```diff
@@ -0,0 +1,15 @@
+export enum Currency {
+  ETH = 'ETH',
+  LOOKS = 'LOOKS',
+  MATIC = 'MATIC',
+}
+
+export const fetchPrice = async (currency: Currency = Currency.ETH): Promise<number | undefined> => {
+  try {
+    const response = await fetch(`https://api.coinbase.com/v2/exchange-rates?currency=${currency}`)
```

Comment:
I think this is fine, but it seems strange to go to coinbase for this price when the global app checks it from uniswap.

Reply:
This is how we retrieved current price on Genie. For the sake of V0 and the scope of this ticket would i be fine if we keep this and add it as tech debt to replace and refactor where it is used?

Reply:
yes - you should make a low-priority jira ticket to document the tech debt (same with i18n work).